build-system/common/check-package-manager.js

```diff
@@ -90,6 +90,7 @@ function yellow(text) {
  * package manager being used is determined.
  **/
 function ensureNpm() {
+  // @ts-ignore: Object is possibly 'undefined'.
```

Comment:
This file is invoked by NPM during `npm install` and is strictly speaking not a part of our `nodejs` build system. Any way we can avoid this annotation?

Reply:
In that case I'll exclude the file in the tsconfig.